src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -0,0 +1,562 @@
+import { ethers } from 'ethers';
+import debounce from 'debounce-promise';
+
+// Types
+import { IPlrDaoStakingMembershipBlock } from '../../types/transactionBlock';
+
+// Components
+import { useEtherspot, useTransactionBuilder } from '../../hooks';
+import Text from '../Text/Text';
+import AccountSwitchInput from '../AccountSwitchInput';
+import NetworkAssetSelectInput from '../NetworkAssetSelectInput';
+import TextInput from '../TextInput';
+import SelectInput, { SelectOption } from '../SelectInput/SelectInput';
+import { RoundedImage } from '../Image';
+
+// providers
+import { IAssetWithBalance } from '../../providers/EtherspotContextProvider';
+
+// utils
+import { formatAmountDisplay, formatMaxAmount } from '../../utils/common';
+import { addressesEqual, isValidEthereumAddress, isValidAmount } from '../../utils/validation';
+import { Chain, supportedChains, plrDaoAsset, plrDaoMemberNFT } from '../../utils/chain';
+import { swapServiceIdToDetails } from '../../utils/swap';
+
+//constants
+import { DestinationWalletEnum } from '../../enums/wallet.enum';
+
+export interface IPlrDaoTransactionBlockValues {
+  hasEnoughPLR: boolean;
+  fromChainId?: number;
+  fromAssetAddress?: string;
+  fromAssetIconUrl?: string;
+  fromAsset?: IAssetWithBalance;
+  selectedAsset?: IAssetWithBalance | null;
+  toAsset?: TokenListToken;
+  fromAssetDecimals?: number;
+  fromAssetSymbol?: string;
+  amount?: string;
+  accountType: AccountTypes;
+  receiverAddress?: string;
+  offer?: ExchangeOffer;
+}
+
+const Title = styled.h3`
+  margin: 0 0 18px;
+  padding: 0;
+  font-size: 16px;
+  color: ${({ theme }) => theme.color.text.cardTitle};
+  font-family: 'PTRootUIWebBold', sans-serif;
+`;
+
+const WalletReceiveWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const OfferDetails = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-family: 'PTRootUIWebMedium', sans-serif;
+`;
+
+const Container = styled.div`
+  align-items: center;
+  font-family: 'PTRootUIWebMedium', sans-serif;
+  background: ${({ theme }) => theme.color.background.tokenBalanceContainer};
+  color: ${({ theme }) => theme.color.text.tokenBalance};
+  padding: 16px;
+  margin: 5px;
+  border-image: linear-gradient(#346ecd, #cd34a2) 30;
+  border-width: 2px;
+  border-style: solid;
+`;
+
+const Value = styled.div`
+  font-family: 'PTRootUIWebMedium', sans-serif;
+  color: ${({ theme }) => theme.color.text.tokenValue};
+  display: contents;
+`;
+
+const Total = styled.div`
+  font-family: 'PTRootUIWebMedium', sans-serif;
+  color: ${({ theme }) => theme.color.text.tokenTotal};
+  display: contents;
+  font-weight: bold;
+`;
+
+const HorizontalLine = styled.div`
+  margin: 9px 0;
+  width: 100%;
+  height: 2px;
+  ${({ theme }) => theme.color.text.tokenTotal}
+  background: ${({ theme }) =>
+    `linear-gradient(90deg, ${theme.color.background.horizontalLineFromColor} , ${theme.color.background.horizontalLineToColor})`};
```

Comment:
I think here instead of keeping two colors we can have a single color variable something like
```
background: ${({ theme }) => theme.color.horizontalLineColor };
```
Reason being in this case if people want they can also add linear gradient , radial gradient and a single color too.